core/tauri-runtime-wry/src/lib.rs

```diff
@@ -2704,6 +2707,29 @@ pub fn center_window(window: &Window, window_size: TaoPhysicalSize<u32>) -> Resu
   }
 }
 
+fn parse_proxy(url: &Url) -> Result<ProxyConfig> {
```

Comment:
"parse_proxy_url" maybe better than "parse_proxy".

In many cases of this PR, I feel that "Proxy Url" is more suitable than just "Proxy".

Reply:
I agree